api/src/main/java/graphql/nadel/compat/ExecutableNormalizedOperationToAstCompiler.java

```diff
@@ -180,4 +182,15 @@ private static SelectionSet selectionSet(List<Field> fields) {
             return argValue(executableNormalizedField, argName, normalizedInputValue.getValue(), variableAccumulator);
         }
     }
+
+    private static boolean isConditionalCompat(ExecutableNormalizedField nf, GraphQLSchema schema) {
```

Comment:
New GraphQL Java version removed the old `isConditional` behavior. So we're adding it back here for the FF.